### ISSUE TITLE
Name field does not persist when validating address

### ISSUE THREAD

**Issue by KidA001** [robhurring/address-validator#7]
```
address = AddressValidator::Address.new(
  name: 'Some Name',
  street1: '501 York Street',
  city: 'San Francisco',
  state: 'CA',
  zip: '94110',
  country: 'US'
)
response = validator.validate(address)

response.valid? # true
response.address.name # nil
repsonse.address.street1 # "501 YORK ST"
```

**Comment by robhurring** [comment]
Hey @KidA001 - I haven't looked at this code in a while, but the documentation says the `CosigneeName` should be coming back in certain cases. I can't trigger it but updated the code to add it into the `Address` if it is in the reponse.

**Comment by robhurring** [comment]
This should be solved with #8 - if you shoot me over a test address I can try to validate it (I couldn't get that behavior to trigger from my tests).
